Python/symtable.c

```diff
@@ -2140,7 +2152,8 @@ symtable_visit_stmt(struct symtable *st, stmt_ty s)
             VISIT_SEQ(st, stmt, s->v.AsyncFor.orelse);
         break;
     }
-    VISIT_QUIT(st, 1);
+    LEAVE_RECURSIVE(st);
```

Comment:
The macro already has a `return 1` (but only in the `symtable.c` file, the other similar macros in other files do not have that return). I'd suggest removing the `return 1` in the macro and keep an explicit on in the code.

Reply:
Thanks, I forget to remove it.